wpilibc/src/main/native/include/frc/SpeedController.h

```diff
@@ -25,6 +28,22 @@ class SpeedController : public PIDOutput {
    */
   virtual void Set(double speed) = 0;
 
+  /**
+   * Sets the voltage output of the SpeedController.  Compensates for
+   * fluctuations in the bus voltage to keep the output voltage constant even
+   * when battery voltage fluctuates - highly useful when the voltage outputs
+   * are "meaningful" (e.g. they come from a feedforward calculation).
+   *
+   * <p>NOTE: This function *must* be called regularly in order for voltage
+   * compensation to work properly - unlike the ordinary set function, it is not
+   * "set it and forget it."
+   *
+   * @param output The voltage to output.
+   */
+  virtual void SetVoltage(units::volt_t output) {
+    Set(output / units::volt_t(RobotController::GetInputVoltage()));
```

Comment:
This should not be inline.  Move this to a .cpp file and that will let you take out the RobotController.h include from the header.